featureflags/graph/graph.py

```diff
@@ -114,6 +115,9 @@ async def root_flags(ctx: dict, options: dict) -> list:
             )
         )
 
+    if flag_name:
+        expr = expr.where(func.lower(Flag.name).like(f"%{flag_name.lower()}%"))
```

Comment:
1. Maybe it makes sence to add separate db index on `name` column.
2. I suggest to use `ilike` instead of `like + func.lower` because if not, it is very likely that index will not be used. More info here: https://use-the-index-luke.com/sql/where-clause/functions/case-insensitive-search